tests/test262_runner.rs

```diff
@@ -187,6 +187,7 @@ impl BaseTest262Runner {
             if !strict {
                 command.arg("--no-strict");
             }
+            command.arg("--nogc");
```

Comment:
**nitpick:** I think the test runner should have a command to opt into extensive GC, where it would not only run GC between script executions, but also between promise jobs.

Reply:
Yup, agreed.

As you pointed out above, right now running between promise jobs would lead to the promises themselves being collected so that should probably be a later addition.

Reply:
I added a `--gc` flag for test262_runner.